ReactNativeClient/lib/models/Setting.js

```diff
@@ -367,6 +367,7 @@ class Setting extends BaseModel {
 			'markdown.plugin.multitable': { value: false, type: Setting.TYPE_BOOL, section: 'plugins', public: true, appTypes: ['mobile', 'desktop'], label: () => _('Enable multimarkdown table extension') },
 			'markdown.plugin.fountain': { value: false, type: Setting.TYPE_BOOL, section: 'plugins', public: true, appTypes: ['mobile', 'desktop'], label: () => _('Enable Fountain syntax support') },
 			'markdown.plugin.mermaid': { value: true, type: Setting.TYPE_BOOL, section: 'plugins', public: true, appTypes: ['mobile', 'desktop'], label: () => _('Enable Mermaid diagrams support') },
+			'markdown.plugin.video': { value: false, type: Setting.TYPE_BOOL, section: 'plugins', public: true, appTypes: ['desktop'], label: () => _('Enable embededing video') },
```

Comment:
You've got a small typo.  Perhaps:  'Enable video embedding'

Reply:
@mic704b thank you for pointing it out, I'm working on the creating a unit test, I'll change this when I'm done with that